plugins/panels/similarity_search/__init__.py

```
@@ -23,6 +23,13 @@
 logger = logging.getLogger(__name__)
 
 
+def _has_manage_permission(ctx):
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `_has_manage_permission`

(ANN202)